channel-utilization/README.md

```diff
@@ -1,26 +1,79 @@
 # Channel Utilization Data
 
-These instructions export forwarding data that has been collected in 
-circuitbreaker.
+These instructions will help you export forwarding data that has been collected in circuitbreaker.
 
-There are two phases in this data collection: 
-1. Pull data from circuitbreaker
-2. Randomization of data to remove sensitive information
+There are two phases in this data collection:
+1. Pull data from circuitbreaker.
+2. Randomize the data to remove sensitive information.
 
-Responses are parsed using [jq](https://jqlang.github.io/jq/) - please 
-open an issue if this requirement is not possible in your production 
-environment!
+Responses are parsed using [jq](https://jqlang.github.io/jq/). Please open an issue if this requirement is not possible in your production environment!
 
-## Run instructions
+## Run Instructions
 
-Run these instructions from inside of the channel-utilization dir:
-`cd channel-utilization`
+1. Clone the git repository of carlaKC/ln-data using:
+   ```sh
+   git clone https://github.com/carlaKC/ln-data/
+   ```
 
-Make sure that the script is executable:
-`chmod +x channel-utilization.sh`
+2. Navigate to the channel-utilization directory:
+   ```sh
+   cd ln-data/channel-utilization
+   ```
 
-Run the script:
-`./channel_utilization.sh`
+3. Run the script:
+   ```sh
+   bash channel_utilization.sh
+   ```
 
-The data will be exported in `fowarding_data.csv`, with timestamps and 
-channel ids randomized to protect the privacy of the collecting node.
+The data will be exported in `forwarding_data.csv`, with timestamps and channel IDs randomized to protect the privacy of the collecting node.
+
+### Addendum for Processing Very Large Circuitbreaker Database with >1M Entries
+
+In case the pull request via the circuitbreaker API does not work because of too many entries or broken database, follow these steps:
+
+1. Copy the circuitbreaker database to the `channel-utilization` directory.
+
+   Example of a copy command depending on where `circuitbreaker.db` and the `channel-utilization` directory are located:
+   ```sh
+   cp /home/circuitbreaker/.circuitbreaker/circuitbreaker.db /home/admin/ln-data/channel-utilization/.
```

Comment:
nit: this is specific to where you cloned `ln-data`

let's update to just dropping it in the working directory? 
`cp /home/circuitbreaker/.circuitbreaker/circuitbreaker.db .`

Would also be great to make a note that the database may be in a difference place on different OS-es (eg mac is: `~/Library/Application\ Support/Circuitbreaker/circuitbreaker.db`)